### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
A few new cal csvs 
Note that 3 sensors are now on Irminger SUMO (last minute swaps).
Also moved NUTNR-M cal csvs (already reviewed/approved on PR#1539) from NUTNRB to NUTNRM folder.  These do not need re-review.

### CODE REVIEW

File: calibration/PHSENE/CGINS-PHSENE-P0172__20240509.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0172,CC_ea434,17372.0,source_file PHSEN-E_SAMI-pH_SN_P0172_Validation_Files_2024-05-09.zip > SAMI_P0172_090524t_ValCert.pdf
+P0172,CC_ea578,94.1,
+P0172,CC_eb434,2284.0,
+P0172,CC_eb578,38676.0,
+P0172,CC_ind_off,0,
+P0172,CC_ind_slp,1,
+P0172,CC_psal,35,Constant
+P0172,CC_sami_bits,14,
```
Comment (cooleyky) [inline calibration/PHSENE/CGINS-PHSENE-P0172__20240509.csv:9]: looks good

File: calibration/PHSENE/CGINS-PHSENE-P0177__20240422.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0177,CC_ea434,17372.0,source_file PHSEN-E_SAMI-pH_SN_P0177_Validation_Files_2024-04-22.zip > SAMI_P0177_220424t_ValCert.pdf
+P0177,CC_ea578,94.1,
+P0177,CC_eb434,2284.0,
+P0177,CC_eb578,38676.0,
+P0177,CC_ind_off,0,
+P0177,CC_ind_slp,1,
+P0177,CC_psal,35,Constant
+P0177,CC_sami_bits,14,
```
Comment (cooleyky) [inline calibration/PHSENE/CGINS-PHSENE-P0177__20240422.csv:9]: looks good

File: calibration/CTDMOG/CGINS-CTDMOG-12311__20240424.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+37-12311,CC_ptca0,5.248915e+005,
+37-12311,CC_ptca1,3.186746e+000,
+37-12311,CC_ptca2,-1.614905e-002,
+37-12311,CC_ptcb0,2.547063e+001,
+37-12311,CC_ptcb1,-3.075000e-003,
+37-12311,CC_ptcb2,0.000000e+000,
+37-12311,CC_ptempa0,-7.059676e+001,
+37-12311,CC_ptempa1,5.219687e-002,
+37-12311,CC_ptempa2,-7.792488e-007,
+37-12311,CC_wbotc,0.000000e+000,
```
Comment (cooleyky) [inline calibration/CTDMOG/CGINS-CTDMOG-12311__20240424.csv:25]: looks good

File: calibration/FLORTD/CGINS-FLORTD-01324__20240422.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+1324,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1324_Calibration_Files_2024-04-22.zip > BBFL2W-1324.dev
+1324,CC_dark_counts_cdom,49,
+1324,CC_dark_counts_chlorophyll_a,50,
+1324,CC_dark_counts_volume_scatter,47.125,
+1324,CC_depolarization_ratio,0.039,Constant
+1324,CC_measurement_wavelength,700,Constant
+1324,CC_scale_factor_cdom,9.099E-02,
+1324,CC_scale_factor_chlorophyll_a,1.209E-02,
+1324,CC_scale_factor_volume_scatter,1.780E-06,
+1324,CC_scattering_angle,124,Constant
```
Comment (cooleyky) [inline calibration/FLORTD/CGINS-FLORTD-01324__20240422.csv:11]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

File: calibration/PHSENE/CGINS-PHSENE-P0172__20240509.csv (hunk 1 above)
Comment (cdobs) [inline calibration/PHSENE/CGINS-PHSENE-P0172__20240509.csv:9]: looks good, cc_sami_bits 14

File: calibration/PHSENE/CGINS-PHSENE-P0177__20240422.csv (hunk 2 above)
Comment (cdobs) [inline calibration/PHSENE/CGINS-PHSENE-P0177__20240422.csv:9]: looks good, cc_sami_bit 14

**Review by cdobs — APPROVED** [review]

**Pull request merged**